File: app/src/main/java/com/demomaster/staticrv/StatickRVActivity.java
package com.demomaster.staticrv;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.Toast;

import com.demomaster.R;

import java.util.ArrayList;
import java.util.List;

public class StatickRVActivity extends AppCompatActivity {

    List<String> nameList = new ArrayList<>();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_statick_rv);
        nameList.add("Test");
        nameList.add("Demo");
        nameList.add("TestDemo");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        nameList.add("DemoTest");
        RecyclerView rv_statick = (RecyclerView) findViewById(R.id.rv_statick);
        LinearLayoutManager layoutManager = new LinearLayoutManager(StatickRVActivity.this);
        rv_statick.setLayoutManager(layoutManager);
        RVAdpter adpter = new RVAdpter(nameList);
        rv_statick.setAdapter(adpter);
        adpter.notifyDataSetChanged();
    }
}

File: app/src/main/java/com/demomaster/kotlin/staticrv/StatickRVActivity.kt
package com.demomaster.kotlin.staticrv

import android.content.Context
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import com.demomaster.databinding.ActivityStatickRvBinding

class StatickRVActivity : AppCompatActivity() {

    private lateinit var binding: ActivityStatickRvBinding
    private var mContext: Context? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityStatickRvBinding.inflate(layoutInflater)
        setContentView(binding.root)

        mContext = this@StatickRVActivity

        setSupportActionBar(binding.toolBare.root)
        if (supportActionBar != null) {
            supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        }
        binding.toolBare.root.setNavigationOnClickListener { finish() }

        val layoutManager = LinearLayoutManager(mContext)
        binding.rvStatic.layoutManager = layoutManager
        val adapter = RVAdapter(addData())
        binding.rvStatic.adapter = adapter
    }

    private fun addData(): ArrayList<String> {
        val nameList = ArrayList<String>()
        nameList.add("Test")
        nameList.add("Demo")
        nameList.add("TestDemo")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        nameList.add("DemoTest")
        return nameList
    }
}